{{ cookiecutter.formal_name }}/{{ cookiecutter.formal_name }}.app/Contents/MacOS/{{ cookiecutter.formal_name }}

```diff
@@ -2,4 +2,4 @@
 CONTENTS_DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )
 export PYTHONPATH="$CONTENTS_DIR/Resources/app":"$CONTENTS_DIR/Resources/app_packages"
 cd "$CONTENTS_DIR/Resources"
-Support/bin/python3 -m {{ cookiecutter.module_name }}
+Support/bin/python3 -m {{ cookiecutter.module_name }} "$@"
```

Comment:
Is the quoting necessary here? If the app has multiple arguments, won't that merge them into a single argument?

Reply:
I do think it's necessary, otherwise a single quoted string with spaces will be passed as multiple arguments. This is shown here: [https://stackoverflow.com/questions/4824590/propagate-all-arguments-in-a-bash-shell-script](https://stackoverflow.com/questions/4824590/propagate-all-arguments-in-a-bash-shell-script).

Reply:
Huh - TIL that `"$@"` accurately quotes individual arguments. Thanks for that link.